django/db/models/sql/compiler.py

```diff
@@ -360,6 +361,20 @@ def compile(self, node, select_format=False):
             return node.output_field.select_format(self, sql, params)
         return sql, params
 
+    def get_combinator_sql(self, combinator):
+        parts = (query.get_compiler(self.using, self.connection).as_sql() for query in self.query.combined_queries)
+        combinator_sql = self.connection.set_operators[combinator]
+        if not self.query.distinct and combinator == UNION:
```

Comment:
You should use `self.connection.set_operators['union']` instead of `UNION` constant.